### ISSUE TITLE
Build on CI broken

### ISSUE THREAD

**Issue by martinvw** [openhab/openhab-webui#48]
@wborn / @openhab/webui-maintainers the build on Jenkins is broken since today, something wrong with the SAT definition. I did not build it locally to validate the error yet.

```
18:02:20 Started by an SCM change
18:02:20 Started by an SCM change
18:02:20 [EnvInject] - Loading node environment variables.
18:02:20 Building in workspace /var/jenkins_home/workspace/openHAB-WebUI
18:02:20 No credentials specified
18:02:20  > git rev-parse --is-inside-work-tree # timeout=10
18:02:20 Fetching changes from the remote Git repository
18:02:20  > git config remote.origin.url https://github.com/openhab/openhab-webui # timeout=10
18:02:20 Fetching upstream changes from https://github.com/openhab/openhab-webui
18:02:20  > git --version # timeout=10
18:02:20  > git fetch --tags --progress https://github.com/openhab/openhab-webui +refs/heads/*:refs/remotes/origin/*
18:02:22  > git rev-parse origin/master^{commit} # timeout=10
18:02:22 Checking out Revision 2878a59ceae55daba1e4c4ba9fd4c84f917c0d45 (origin/master)
18:02:22  > git config core.sparsecheckout # timeout=10
18:02:22  > git checkout -f 2878a59ceae55daba1e4c4ba9fd4c84f917c0d45
18:02:22 Commit message: "Fix Java 11 build (#41)"
18:02:22  > git rev-list --no-walk 249e33b2f88356650db66d85a5aba243fe6176f8 # timeout=10
18:02:22 Cleaning workspace
18:02:22  > git rev-parse --verify HEAD # timeout=10
18:02:22 Resetting working tree
18:02:22  > git reset --hard # timeout=10
18:02:23  > git clean -fdx # timeout=10
18:03:15 Jenkins Artifactory Plugin version: 2.16.2
18:03:16 Parsing POMs
18:03:19 ERROR: Failed to parse POMs
18:03:19 org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 [ERROR] 'build.plugins.plugin.version' for org.openhab.tools.sat:sat-plugin must be a valid version but is '${sat.version}'. @ org.openhab.ui:org.openhab.ui.reactor:2.5.0-SNAPSHOT, /var/jenkins_home/workspace/openHAB-WebUI/pom.xml, line 522, column 24
18:03:19 
18:03:19 	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:364)
18:03:19 	at org.jfrog.build.extractor.maven.ArtifactoryProjectBuilder.build(ArtifactoryProjectBuilder.java:31)
18:03:19 	at hudson.maven.MavenEmbedder.buildProjects(MavenEmbedder.java:361)
18:03:19 	at hudson.maven.MavenEmbedder.readProjects(MavenEmbedder.java:331)
18:03:19 	at hudson.maven.MavenModuleSetBuild$PomParser.invoke(MavenModuleSetBuild.java:1328)
18:03:19 	at hudson.maven.MavenModuleSetBuild$PomParser.invoke(MavenModuleSetBuild.java:1125)
18:03:19 	at hudson.FilePath.act(FilePath.java:1074)
18:03:19 	at hudson.FilePath.act(FilePath.java:1057)
18:03:19 	at hudson.maven.MavenModuleSetBuild$MavenModuleSetBuildExecution.parsePoms(MavenModuleSetBuild.java:986)
18:03:19 	at hudson.maven.MavenModuleSetBuild$MavenModuleSetBuildExecution.doRun(MavenModuleSetBuild.java:691)
18:03:19 	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:504)
18:03:19 	at hudson.model.Run.execute(Run.java:1816)
18:03:19 	at hudson.maven.MavenModuleSetBuild.run(MavenModuleSetBuild.java:543)
18:03:19 	at hudson.model.ResourceController.execute(ResourceController.java:97)
18:03:19 	at hudson.model.Executor.run(Executor.java:429)
18:03:19 [openHAB-WebUI] $ /bin/sh -xe /tmp/jenkins3955556582398135702.sh
18:03:19 + cd /var/jenkins_home/workspace/openHAB-WebUI
18:03:19 + chmod 744 repo/target/publish-bintray.sh
18:03:19 chmod: cannot access 'repo/target/publish-bintray.sh': No such file or directory
18:03:19 Build step 'Execute shell' marked build as failure
18:03:19 [ANALYSIS-COLLECTOR] Computing warning deltas based on reference build #28
18:03:19 Archiving artifacts
18:03:20 Finished: FAILURE

**Comment by wborn** [comment]
SAT worked fine after #36, but it got removed again with #44. So it was readded with #46, but apparently still has this issue @maggu2810 .

**Comment by maggu2810** [comment]
Sorry guys, I am sick and tired these day.
Perhaps it had been a better idea to wait a few days instead of create the PR(s) today.
It seems I forgot to set the property (sat.version).
I am back at home on sunday evening, then I can have a look at again.

**Comment by wborn** [comment]
The new layout is definitely better! I think everything is fixed with #50.

**Comment by wborn** [comment]
The build fix was already merged as part of https://github.com/openhab/openhab-webui/pull/52.
So this issue can be closed @openhab/webui-maintainers.
